data/tx/municipalities/Ron-Nirenberg--1dedd1e4-bd05-4fde-b688-2145b7070bee.yml

```diff
@@ -4,7 +4,7 @@ given_name: Ron
 family_name: Nirenberg
 email: mayor@cityofsanantonio.gov
 roles:
-- end_date: '2023-11-30'
+- end_date: '2025-11-30'
```

Comment:
I can't find if it's actually 11-30 or not, every source I can find just says the year the term ends (2025).